librepcb/firstrunwizard/firstrunwizard.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * LibrePCB - Professional EDA for everyone!
+ * Copyright (C) 2013 Urban Bruhin
+ * http://librepcb.org/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*****************************************************************************************
+ *  Includes
+ ****************************************************************************************/
+
+#include "firstrunwizard.h"
+#include "ui_firstrunwizard.h"
+#include "firstrunwizardpage_welcome.h"
+#include "firstrunwizardpage_workspacepath.h"
+
+/*****************************************************************************************
+ *  Constructors / Destructor
+ ****************************************************************************************/
+
+FirstRunWizard::FirstRunWizard(QWidget *parent) noexcept :
+    QWizard(parent), mUi(new Ui::FirstRunWizard)
```

Comment:
This ``new/delete`` pairs should real`ly be replaced by ``std::unique_ptr<T>``. If this class owns the ``Ui::FirstRunWizard`` a unique pointer is perfectly suited.